app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Category;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        $category = new Category();
+        $category->title = $request->title;
```

Comment:
Since Laravel is using MVC Desing pattern, we stick to the concept that DB Logic Or Eloquent is managed by a Model Class, Please transfer these code to the Category Model.